Cargo.toml

```diff
@@ -47,7 +47,7 @@ missing_safety_doc = "warn"
 [workspace.package]
 version = "0.4.0"
 authors = [
-    "Lily Lyons <lily@nowaffles.com>",
```

Comment:
@white-axe While I'm at it, do you want to be added to the crate authors?

Reply:
Sure, add me as "Hao Liu \<whiteaxe@tuta.io\>"